chainstate/storage/src/internal/test.rs

```diff
@@ -353,11 +388,12 @@ fn undo_test(#[case] seed: Seed) {
 #[case(Seed::from_entropy())]
 fn utxo_db_impl_test(#[case] seed: Seed) {
     let mut rng = make_seedable_rng(seed);
-    let mut store = crate::inmemory::Store::new_empty().expect("should create a store");
-    store
+    let store = crate::inmemory::Store::new_empty().expect("should create a store");
+    let mut db_tx = store.transaction_rw(None).expect("should create a transaction");
+    db_tx
         .set_best_block_for_utxos(&H256::random_using(&mut rng).into())
         .expect("Setting best block cannot fail");
-    let mut db_interface = utxo::UtxosDB::new(&mut store);
+    let mut db_interface = utxo::UtxosDB::new(&mut db_tx);
```

Comment:
Looks like this whole test now runs in a rw transaction which at the end goes out of scope so nothing is written to the storage. While it seems fine in this test, it's a change in behaviour compared to what was there before.